<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QMainWindow" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton { border: 1px solid black }
QPushButton :hover { border: 1px solid green }

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>261</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame
{
   border: 1px solid black;
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListWidget" name="listWidget_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>430</y>
       <width>141</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>390</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Friends</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Rank</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Points</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ArchiveButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>240</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Archive</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AddFriendButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>320</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Add friend</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SignOutButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>160</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Sign out</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Test</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="Img">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>265</width>
      <height>651</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/Img/Img/LogoJpeg.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Img2">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>0</y>
      <width>265</width>
      <height>651</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/Img/Img/LogoJpeg2.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Img3">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>0</y>
      <width>265</width>
      <height>651</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/Img/Img/LogoJpeg.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>297</x>
      <y>20</y>
      <width>721</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame
{
	background-color: white;
    border: 1px solid black;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {border: 1px solid black; } 
QPushButton:hover { border: 1px solid darkgreen;}</string>
     </property>
     <property name="text">
      <string>Add task</string>
     </property>
    </widget>
    <widget class="QLabel" name="Categorylabel">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	border: none;
}</string>
     </property>
     <property name="text">
      <string>All Categories</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>20</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {border: 1px solid black; } 
QPushButton:hover { border: 1px solid darkgreen;}</string>
     </property>
     <property name="text">
      <string>Add category</string>
     </property>
    </widget>
    <widget class="QListWidget" name="ToDo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>224</width>
       <height>491</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="InProgress">
     <property name="geometry">
      <rect>
       <x>243</x>
       <y>100</y>
       <width>234</width>
       <height>491</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="Done">
     <property name="geometry">
      <rect>
       <x>476</x>
       <y>100</y>
       <width>224</width>
       <height>491</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="ToDolabel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	border: none;
}</string>
     </property>
     <property name="text">
      <string>TODO</string>
     </property>
    </widget>
    <widget class="QLabel" name="InProgresslabel">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	border: none;
}</string>
     </property>
     <property name="text">
      <string>IN PROGRESS</string>
     </property>
    </widget>
    <widget class="QLabel" name="Donelabel">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>80</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	border: none;
}</string>
     </property>
     <property name="text">
      <string>DONE</string>
     </property>
    </widget>
   </widget>
   <zorder>Img</zorder>
   <zorder>Img3</zorder>
   <zorder>Img2</zorder>
   <zorder>frame</zorder>
   <zorder>frame_2</zorder>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
